gslib/tests/test_rpo.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration tests for rpo command."""
+
+from __future__ import absolute_import
+
+import textwrap
+
+from gslib.commands.rpo import RpoCommand
+from gslib.exception import CommandException
+from gslib.gcs_json_api import GcsJsonApi
+from gslib.storage_url import StorageUrlFromString
+import gslib.tests.testcase as testcase
+from gslib.tests.testcase.integration_testcase import SkipForGS
+from gslib.tests.testcase.integration_testcase import SkipForJSON
+from gslib.tests.testcase.integration_testcase import SkipForXML
+from gslib.tests.util import ObjectToURI as suri
+from gslib.tests.util import SetBotoConfigForTest
+
+from six import add_move, MovedModule
+
+add_move(MovedModule('mock', 'mock', 'unittest.mock'))
+from six.moves import mock
+
+
+class TestLsUnit(testcase.GsUtilUnitTestCase):
+
+  def test_get_for_multiple_bucket_calls_api(self):
+    bucket_uri1 = self.CreateBucket(bucket_name='rpofoo')
+    bucket_uri2 = self.CreateBucket(bucket_name='rpobar')
+    stdout = self.RunCommand(
+        'rpo',
+        ['get', suri(bucket_uri1), suri(bucket_uri2)],
+        return_stdout=True)
+    expected_string = textwrap.dedent("""\
+      gs://rpofoo: None
+      gs://rpobar: None
+      """)
+    self.assertEqual(expected_string, stdout)
+
+  def test_get_with_wildcard(self):
+    self.CreateBucket(bucket_name='boo1')
+    self.CreateBucket(bucket_name='boo2')
+    stdout = self.RunCommand('rpo', ['get', 'gs://boo*'], return_stdout=True)
+    actual = '\n'.join(sorted(stdout.strip().split('\n')))
```

Comment:
Is the sort necessary?

Reply:
yeah, the output can be in random order.